src/templates/tournaments/tournament_archived_list.html

```diff
@@ -0,0 +1,131 @@
+{% extends "base.html" %}
+
+{% load static %}
+
+{% block extrahead %}
+  <link rel="stylesheet"
+        type="text/css"
+        href="{% static 'tournament_styles.css' %}" />
+{% endblock extrahead %}
+{% block content %}
+  <h1>Tournaments</h1>
```

Comment:
I wonder if we should change this header to be "Archived Tournaments" or something similar. Right now, I almost got lost as to whether I was on the Tournaments list view or the Archived Tournaments list view.

Reply:
Thanks for the comment! I will change this.